jbmc/unit/java-testing-utils/require_type.h

```diff
@@ -33,11 +33,14 @@ struct_typet::componentt require_component(
   const irep_idt &component_name);
 
 code_typet require_code(const typet &type);
+java_method_typet require_java_method(const typet &type);
 
-code_typet::parametert
+java_method_typet::parametert
```

Comment:
Sorry to be a pain - this wants to be left as code_typet - but I trust you to apply this!